docs/reference/action_list.rst

```diff
@@ -248,6 +248,17 @@ You can customize the list query thanks to the ``configureQuery`` method::
 Customizing the sort order
 --------------------------
 
+Default ordering in the list view
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default ordering may be defined in:
+
+1. ``$datagridValues`` property (of your Admin class)
```

Comment:
`getDefaultSortValues` should be in the list. And datagridValues need to be removed in the next major.